modules/uncertainty/R/get.parameter.samples.R

```diff
@@ -58,9 +58,14 @@ get.parameter.samples <- function(settings,
     ## Load posteriors
     if (!is.na(posterior.files[i])) {
       # Load specified file
-      load(posterior.files[i], envir = distns)
-      if (is.null(distns$prior.distns) & !is.null(distns$post.distns)) {
-        distns$prior.distns <- distns$post.distns
+      load(posterior.files[i])
+      fname <- file.path(outdirs[i], "post.distns.Rdata")
+      if (file.exists(fname)) {
+        load(fname)
+        prior.distns <- post.distns
+      }
+      if (exists("trait.mcmc") & exists("post.distns")) {
+        post.distns <- post.distns[!(row.names(post.distns) %in% names(trait.mcmc)),]
```

Comment:
This appears to be undoing recent changes from #3220 -- is that intentional?

Reply:
This function was adapted from Alexis Helgeson's codes to read "basal soil respiration rate" from "trait.mcmc.Rdata" as we found basal soil respiration rate seems too high from "post.distns.Rdata" for NEON sites. I am not sure whether she put forward a PR for this change. But maybe we can revert it for the develop branch as the changes are specific for our study?

Reply:
@Qianyuxuan so @infotroph isn't objecting to using trait.mcmc or post.distns, but that when you resolved you merge conflict you undid the effort to load the files into a specific envir and then use that envir name (distns). I think you can easily adapt your code to include the previous changes.

Reply:
Yes, correct -- loading files into a specific envir is the important part. 
Also, to make sure I'm reading right: Am I correct that all the changes in this file are specific to basal soil respiration, they're simply making sure trait.mcmc and post.distns are available to be processed elsewhere?

Reply:
@Qianyuxuan you still need to fix this change before this PR can go in.